libmscore/barline.cpp

```diff
@@ -474,8 +474,8 @@ void BarLine::drawDots(QPainter* painter, qreal x) const
       else {
             const StaffType* st = staffType();
 
-            //workaround to make new Bravura and Petaluma font work correctly with repeatDots
-            qreal offset = (score()->scoreFont()->name() == "Bravura" || score()->scoreFont()->name() == "Petaluma") ? 0 : 0.5 * score()->spatium() * mag();
+            //workaround to make new Bravura, Petaluma and Leland font work correctly with repeatDots
+            qreal offset = (score()->scoreFont()->name() == "Bravura" || score()->scoreFont()->name() == "Petaluma" || score()->scoreFont()->name() == "Leland") ? 0 : 0.5 * score()->spatium() * mag();
```

Comment:
Maybe we can change the order, Leland (the default) first, Pelatuma (the least likely) last? Or would that be 'micro-optimazation'?

Reply:
yeah, why not